cmd/dex/config.go

```diff
@@ -119,6 +125,46 @@ type GRPC struct {
 	TLSClientCA string `json:"tlsClientCA"`
 }
 
+type Userinfo struct {
+	Type   string         `json:"type"`
+	Config UserinfoConfig `json:"config"`
+}
+
+type UserinfoConfig interface {
+	Open(logrus.FieldLogger) (userinfo.Userinfo, error)
+}
+
+var userinfoAdapters = map[string]func() UserinfoConfig{
```

Comment:
drdAdapters as var name?

Reply:
disagree, as this is the generic part of an abstracted userinfo adapter, rather than binding this to be a DRD only